test/system/src/main/java/io/pravega/test/system/framework/kubernetes/K8sClient.java

```diff
@@ -666,31 +666,63 @@ public void deletePVC(String namespace, String name) {
      * @return A future which completes once the number of running pods matches the given criteria.
      */
     public CompletableFuture<Void> waitUntilPodIsRunning(String namespace, String labelName, String labelValue, int expectedPodCount) {
+        return waitUntilPodIsRunningRetries(namespace, labelName, labelValue, expectedPodCount, Byte.MAX_VALUE);
```

Comment:
Why `Byte.MAX_VALUE` retries? This looks totally arbitrary, but at least should be documented as a constant.

Reply:
Correct, that was an arbitrarily chosen value. Will create another constant for it.